_inc/lib/class.jetpack-service-helper.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+class Jetpack_Service_Helper {
```

Comment:
We call those External Services or Keyring Services on calypso and wpcom. I think we should use that naming convention as well, Service only is a bit too broad.